ci_framework/roles/cifmw_ceph_spec/handlers/main.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
`handlers` directory is emtpy. We can drop it.

Reply:
done